src/openforms/submissions/tests/test_submission_status.py

```diff
@@ -277,6 +278,62 @@ def test_appointment_user_error(self):
             self.assertEqual(response_data["confirmationPageContent"], "")
             self.assertEqual(response_data["reportDownloadUrl"], "")
 
+    def test_displaying_main_website_link_returns_main_link_through_api(self):
+        config = GlobalConfiguration.get_solo()
```

Comment:
I thought that it is better to use the patch for the get_solo, because it can affect the isolation of the tests?

Reply:
yes, 100% correct - I didn't look at the tests in depth anymore and part of this PR was originally done by someone else :)